src/api/selection.js

```diff
@@ -9,8 +9,21 @@ define(function () {
     // find the parent document or document fragment
     if( rootDoc.compareDocumentPosition(scribe.el) & Node.DOCUMENT_POSITION_DISCONNECTED ) {
       var currentElement = scribe.el.parentNode;
-      while(currentElement && nodeHelpers.isFragment(currentElement)) {
-        currentElement = currentElement.parentNode;
+      // Old browsers might not know about `:host` without polyfill, lets catch exception if it is the case
```

Comment:
Yes, exactly. So this is the selection module so it feels like the wrong place for a Shadow DOM check, putting it in a Shadow DOM module and then importing it would make it easier to maintain.